src/main/java/stirling/software/SPDF/config/AppUpdateShowService.java

```diff
@@ -0,0 +1,62 @@
+package stirling.software.SPDF.config;
+
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import stirling.software.SPDF.model.ApplicationProperties;
+import stirling.software.SPDF.model.User;
+import stirling.software.SPDF.repository.UserRepository;
+
+@Configuration
+public class AppUpdateShowService {
+
+    @Autowired private UserRepository userRepository;
+    @Autowired private ApplicationProperties applicationProperties;
+
+    @Bean
+    public AppUpdateService showUpdate() {
+        return new AppUpdateService(userRepository, applicationProperties);
+    }
+}
+
+class AppUpdateService {
+
+    private UserRepository userRepository;
+    private ApplicationProperties applicationProperties;
+
+    private static final Logger logger = LoggerFactory.getLogger(AppUpdateShowService.class);
+
+    public AppUpdateService(
+            UserRepository userRepository, ApplicationProperties applicationProperties) {
+        this.userRepository = userRepository;
+        this.applicationProperties = applicationProperties;
+    }
+
+    public boolean isShow() {
+        boolean showUpdate = applicationProperties.getSystem().getShowUpdate();
```

Comment:
Maybe we should have a check for this and return it immediately since it's used in every branch?